### PR TITLE
CGE-50 product page

### PR DESCRIPTION
**Pull request by SergeyCheremisin**
Added functionality to product page and images, refactored the code

### CODE REVIEW

**Review by rpkyrych — APPROVED** [review]

File: src/main/resources/react/src/REST/Service/networkProvider.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { getStorageUser, getStorageLang } from '../../redux/Utils';
 
 
 const instance = axios.create( {
-	baseURL: '/',
+	baseURL: 'http://localhost:8080/',
```
Comment (Wolshebnik) [inline src/main/resources/react/src/REST/Service/networkProvider.js:6]: You will fall here
url must be ''/''

File: src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/NoPhoto.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+const NoPhoto = ( { noPhoto = 'noPhoto' , noPhotoImg = 'noPhotoImg' } ) => {
+  return (
+    <div className = { noPhoto }>
+      <svg width = "299" height = "281" viewBox = "0 0 299 281" fill = "none" xmlns = "http://www.w3.org/2000/svg"
+           className = { noPhotoImg }>
+        <path
+          d = "M264.959 0.349976H33.787C15.153 0.349976 0 15.498 0 34.154V246.615C0 265.249 15.153 280.381 33.787 280.381H264.958C283.592 280.381 298.729 265.249 298.729 246.615V34.154C298.73 15.498 283.593 0.349976 264.959 0.349976ZM193.174 50.623C211.194 50.623 225.808 65.238 225.808 83.257C225.808 101.276 211.193 115.891 193.174 115.891C175.149 115.891 160.54 101.276 160.54 83.257C160.54 65.238 175.149 50.623 193.174 50.623ZM254.363 249.149H149.362H49.039C40.026 249.149 36.012 242.628 40.075 234.583L96.081 123.653C100.139 115.609 107.873 114.891 113.35 122.048L169.666 195.644C175.143 202.802 184.716 203.411 191.052 196.998L204.829 183.047C211.16 176.634 220.488 177.428 225.655 184.809L261.33 235.768C266.487 243.16 263.376 249.149 254.363 249.149Z"
+          fill = "#EEEEEE"/>
+      </svg>
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/NoPhoto.js:11]: Can you save a svg in the folder img?

File: src/main/resources/react/src/REST/Service/networkProvider.js (hunk 1 above)
Comment (SergeyCheremisin) [reply src/main/resources/react/src/REST/Service/networkProvider.js]: thanks, fixed

File: src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/NoPhoto.js (hunk 2 above)
Comment (SergeyCheremisin) [reply src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/NoPhoto.js:11]: Unfortunately...

**Review by Jack11M — APPROVED** [review]

File: src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/NoPhoto.js (hunk 2 above)
Comment (SergeyCheremisin) [reply src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/NoPhoto.js]: remade this file, thanks

File: src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/ProductPhotoCarousel.scss [hunk 3]
```diff
@@ -10,9 +10,12 @@
...
-  height: 680px;
+.productPhotoSlideBig {
+
+  img {
+    width: 600px;
+    height: 680px;
+  }
 }
 
 .sliderPosition {
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/ProductPhotoCarousel.scss:21]: Do you need this?

Comment (SergeyCheremisin) [reply src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/ProductPhotoCarousel.scss:21]: Yes, this size of big image

**Pull request merged**